### ISSUE TITLE
Error at setup.py => Install

### ISSUE THREAD

**Issue by the-Blue-Falcon** [theInfectedDrake/TIDoS-Framework#1]
While running `setup.py` to install TIDoS, I encountered the following error:
```
Traceback on <module> line 37
pip.main(['install','requests','scapy','urllib2'])
AttributeError: pip has no attribute main
```
Any idea why this is happening? 

P.S. - I don't know what happened to the earlier version of TIDoS-Framework on your previous account `the-Infected-Drake`. But yeah I have found you and will continue to emulate you. Big fan here!

**Comment by theinfecteddrake** [comment]
Hmm, seems there is some real issue regarding my code in `setup.py`. I will make a update rightaway.

**Comment by the-Blue-Falcon** [comment]
Yeah the bug fixed! Thank you for the attention and help. You're awesome !

**Comment by theinfecteddrake** [comment]
Seems like the bug was fixed. Thanks for the issue. 👍

**Comment by theInfectedDrake** [comment]
Have to put up a pip.main(). Needs help.

**Comment by theInfectedDrake** [comment]
Fixing this bug is over...
